scvi/dataset/smfish.py

```diff
@@ -63,9 +64,9 @@ def preprocess(self):
                     new_labels.append(5)
 
             select = np.array(to_keep)
-            labels, cell_types = np.array(new_labels), np.array(ds.ca['ClusterName'])
+            labels = np.array(new_labels)
             labels = np.reshape(labels, (labels.shape[0], 1))
-            cell_types = np.reshape(cell_types, (cell_types.shape[0], 1))[select]
```

Comment:
cell_types is usually used in the codebase for the name of th ecell types and not the labels. Was this a glitch ?

Reply:
Not sure I get your comment. Cell types in smfish were wrong because they were not the names indeed but an array for each cell. I am removing this wrong line in the commit.

Reply:
yes that's my point, ok it was a glitch